az-digital/az_quickstart
# #2321: Add page and block template workarounds for empty sidebars issue affecting recent Menu Block releases.

*Note this PR targets the dependabot branch for updating menu_block to the latest version (and should hopefully un-block that PR).*

This PR adds a workaround to the page template (inspired by https://www.drupal.org/project/menu_block/issues/3271218#comment-14499729) to deal with the empty sidebar issue that's prevented us from updating to the latest Menu Block release(s).

## Description


## Related issues




Provides workaround for #2321

## How to test



Shared by @bberndt-uaz in #2320:
>Tested in Probo just in case: I created a new system menu, added the menu as a block to the Sidebar First region, and disabled the default Quickstart sidebar menu block. With the Initial visibility level set to 1, the new sidebar block appears on all pages. With the Initial visibility level set to 2, the sidebar properly appears only on the page which is included in the new menu.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
